cfn.template.yaml

```diff
@@ -259,8 +264,8 @@ Resources:
       VpcId: !Ref FrontendVpcId
       SecurityGroupIngress:
       - IpProtocol: tcp
-        FromPort: '80'
-        ToPort: '80'
+        FromPort: '443'
+        ToPort: '443'
```

Comment:
Why not to 80?

Reply:
@jeteve You mean why not listen on port 80, or why not have `ToPort` as 80?

In the latter case; the ingress rules take ranges, and this restricts it to one port only. In the former case, we only want to run the server over TLS.